lib/realtime_web/channels/realtime_channel.ex

```diff
@@ -249,8 +275,6 @@ defmodule RealtimeWeb.RealtimeChannel do
 
   @impl true
   def handle_info(%{event: type, payload: payload}, socket) do
-    socket = count(socket)
```

Comment:
taking this out b/c we should count only when we are absolutely sure we want to.

Reply:
I don't know, this is a catch all for a reason.

I would remove the whole function then and make sure we specifically handle all `event` types. Which, I think we are.

Reply:
yep, removing this.